homeassistant/components/lg_thinq/sensor.py

```diff
@@ -255,9 +267,77 @@
         translation_key=ThinQProperty.WATER_TYPE,
     ),
 }
+ELAPSED_DAY_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.ELAPSED_DAY_STATE: SensorEntityDescription(
+        key=ThinQProperty.ELAPSED_DAY_STATE,
+        native_unit_of_measurement=UnitOfTime.DAYS,
+        translation_key=ThinQProperty.ELAPSED_DAY_STATE,
+    ),
+    ThinQProperty.ELAPSED_DAY_TOTAL: SensorEntityDescription(
+        key=ThinQProperty.ELAPSED_DAY_TOTAL,
+        native_unit_of_measurement=UnitOfTime.DAYS,
+        translation_key=ThinQProperty.ELAPSED_DAY_TOTAL,
+    ),
+}
+TIME_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    TimerProperty.LIGHT_START: SensorEntityDescription(
+        key=TimerProperty.LIGHT_START,
+        translation_key=TimerProperty.LIGHT_START,
+    ),
+    TimerProperty.ABSOLUTE_TO_START: SensorEntityDescription(
+        key=TimerProperty.ABSOLUTE_TO_START,
+        translation_key=TimerProperty.ABSOLUTE_TO_START,
+    ),
+    TimerProperty.ABSOLUTE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.ABSOLUTE_TO_STOP,
+        translation_key=TimerProperty.ABSOLUTE_TO_STOP,
+    ),
+    TimerProperty.TOTAL: SensorEntityDescription(
+        key=TimerProperty.TOTAL,
+        translation_key=TimerProperty.TOTAL,
+    ),
+    TimerProperty.RELATIVE_TO_START: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_START,
+        translation_key=TimerProperty.RELATIVE_TO_START,
+    ),
+    TimerProperty.RELATIVE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_STOP,
+        translation_key=TimerProperty.RELATIVE_TO_STOP,
+    ),
+    TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP,
+        translation_key=TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP,
+    ),
+}
```

Comment:
We still want to get a proper device class here instead of just showing `"13:14:00"` like we see in the `.ambr` file. What are they exactly describing? The duration until something happens? How often do these change?

Reply:
@joostlek 
LIGHT_START, ABSOLUTE_TO_START, ABSOLUTE_TO_STOP should express the time set by the user, but there is no suitable device_class. So, we want to implement it in value.strftime("%I:%M %p").
Do you agree below's time format?

| Time / Timer | Name | When | Data | Format (device_class) | Show |
|--------|--------|--------|--------|--------|--------|
| Time | LIGHT_START | only when user's setting changed | time | %I:%M %p | 12:30 PM |
| Time | ABSOLUTE_TO_START | only when user's setting changed | time | %I:%M %p | 12:30 PM |
| Time  | ABSOLUTE_TO_STOP | only when user's setting changed | time | %I:%M %p | 12:30 PM |
| Time  | TOTAL | user's action| time | min (DURATION) | 120 min |
| Time  | RELATIVE_TO_START | only when user's setting changed | time | min (DURATION) | 120 min |
| Time  | RELATIVE_TO_STOP | only when user's setting changed | time | min (DURATION) | 120 min |
| Time  | SLEEP_TIMER_RELATIVE_TO_STOP | only when user's setting changed | time | min (DURATION) | 120 min |